c++/src/capnp/rpc-twoparty.c++

```diff
@@ -117,7 +124,8 @@ class TwoPartyVatNetwork::OutgoingMessageImpl final
 public:
   OutgoingMessageImpl(TwoPartyVatNetwork& network, uint firstSegmentWordSize)
       : network(network),
-        message(firstSegmentWordSize == 0 ? SUGGESTED_FIRST_SEGMENT_WORDS : firstSegmentWordSize) {}
+        message(firstSegmentWordSize == 0 ? SUGGESTED_FIRST_SEGMENT_WORDS : firstSegmentWordSize),
+        created(network.clock.now()) {}
```

Comment:
It seems slightly more fair to set the timestamp based on when `send()` was called (since any waiting between the creation of the message and the calling of `send()` doesn't seem like the fault of the network), but I don't have any reason to believe it would make a particular difference in practice.

Reply:
Probably doesn't matter in practice, but it's not hard to fix so I will

Reply:
This could matter in practice. I have written code before that begins constructing a request, then waits for some further event before sending it.